docs/design/fabric-v2+/fpc-validation.puml

```diff
@@ -52,88 +52,107 @@ group endorsement
     end ref
     return proposal response
 
+    alt FPC "light"
+        User1 -> ECC1 ++: invoke "validateEnclaveEndorsement" (proposal response)
+            group enclave endorsement validation
+                ECC1 -> ECC1 : <args, read/writeset, result> <- extract from transaction
+                ECC1 -> ECC1 : <Enclave_VK, Sig_Enclave, ECC_result> <- extract from result
```

Comment:
Should we replace `proposal response` with `ccresp_msg` here? 

As mentioned before, without the "full" proposal response from the actual chaincode invocation we are missing the args. I would just add the entire (signed and serialized) proposal to `ccresp_msp` and we have everything we need.

Reply:
well, seeing it now here you actually _would_ have received everything according to the UMLs. But it wouldn't have matched interfaces.md and having `FPC_Stub` add it to `ChaincodeResponseMessage` rather than having the client send it separately in the validation transaction is certainly the easier way to solve it.